tests/contrib/anthropic/test_anthropic_llmobs.py

```diff
@@ -346,7 +361,7 @@ def test_tools_sync(self, anthropic, ddtrace_global_config, mock_llmobs_writer,
                         "content": WEATHER_OUTPUT_MESSAGE_1,
                         "role": "assistant",
                     },
-                    {"content": WEATHER_OUTPUT_MESSAGE_2, "role": "assistant"},
+                    {"content": WEATHER_INPUT_MESSAGE_2, "role": "assistant"},
```

Comment:
Why are we still expecting the old placeholder message value in the input? Are we not expecting the new tool call format in the input as well?